.travis.yml

```diff
@@ -8,6 +8,12 @@ before_install:
   - sudo apt-get update -q
   - sudo apt-get install glide -y
   - mkdir -p $GOPATH/bin
+  - make deps
+
+script:
+  - make check-style
+  - make test
```

Comment:
Because `make coverage` includes the task executing `go test`, is not `make test` necessary?